modules/http4s/test/src/smithy4s/http4s/NullsAndDefaultEncodingSuite.scala

```diff
@@ -0,0 +1,235 @@
+package smithy4s.http4s
+
+import weaver._
+import org.http4s._
+import org.http4s.implicits._
+import cats.effect.IO
+import smithy4s.example.ServiceWithNullsAndDefaults
+import smithy4s.example.OperationOutput
+import io.circe.Json
+import org.typelevel.ci.CIString
+import org.typelevel.ci._
+import org.http4s.circe.CirceInstances
+import smithy4s.Document
+import org.http4s.client.Client
+import smithy4s.example.OperationInput
+import cats.effect.kernel.Deferred
+
+object NullsAndDefaultEncodingSuite extends SimpleIOSuite with CirceInstances {
```

Comment:
**question**: wasn't sure if this is a proper way, and place (`http4s` module) to test it.